client/src/main/java/io/pravega/keycloak/client/KeycloakAuthzClient.java

```diff
@@ -208,7 +264,8 @@ public KeycloakAuthzClient build() {
 
             // create the Keycloak authz client
             ClientAuthenticator authenticator = createClientAuthenticator(configuration.getResource(), (String) configuration.getCredentials().get("secret"));
-            AuthzClient client = clientSupplier.apply(configuration, authenticator);
+            AuthzClient client = Retry.withExpBackoff(httpInitialDelayMs, 2, httpMaxRetries)
```

Comment:
Had to add a retry here as well, as the creation of the client actually reaches out to Keycloak to download endpoints based on the discovery URL.  This too can fail if there are connection problems.